src/tools/batchrun/main.cpp

```diff
@@ -293,7 +293,7 @@ int main(int argc, char* argv[])
             if (optForceParallel)
                 cmd << " --force-parallel=" << *optForceParallel;
 	    if (ortoolsUsed)
- 	        cmd << " --ortools-solver=" << ortoolsSolver;
+ 	        cmd << " --use-ortools --ortools-solver=" << ortoolsSolver;
 
```

Comment:
We should check that ortools solver is not empty, otherwise the command will not be valid